proto/p4/p4runtime.proto

```diff
@@ -136,6 +136,14 @@ message TableEntry {
   // this is stored, as long as it is returned with the rest of the entry in
   // a Read RPC.
   uint64 controller_metadata = 5;
+  // meter_config and counter_data are convenience fields that enable the
+  // controller to configure the direct resources associated with the table at
+  // the same time as a match-action entry is inserted or modified. When
+  // updating a table entry, leaving these fields unset means that the direct
+  // resources (if any) will not be updated. When reading a table entry, the
+  // values associated with the direct resources (if any) will be returned.
```

Comment:
We may need to clarify @yyetim text then. Let's say my table has a direct counter associated with it. If I add a table entry with a Write request (empty CounterData, so the counter's initial value is 0), then performs a Read on this entry, assuming some traffic went through and my flow entry was hit, then the counter value will not be 0 anymore, and obviously the message returned by Read is different from the message of the original Write.
@yyetim @wmohsin any thoughts on this?